### ISSUE TITLE
Error when enabling multipleVersions

### ISSUE THREAD

**Issue by mitar** [mjeanroy/rollup-plugin-license#1682]
I am using Vite 5.0.10 with rollup-plugin-license 3.2.0. The example config works, but for `multipleVersions` it complains:

```
[rollup-plugin-license] -- Unknown property: "thirdParty.multipleVersions", allowed options are: sourcemap, debug, cwd, banner, thirdParty.
```

It works for all other config inside `thirdParty` object. Only if `multipleVersions` is set it complains. I do not know why.

**Comment by mitar** [comment]
I think the issue here is that [schema defined here](https://github.com/mjeanroy/rollup-plugin-license/blob/master/src/license-plugin-option.js#L61) is missing `multipleVersions`.

**Comment by mjeanroy** [comment]
Hi @mitar,

Thanks for the report and the analysis.
I just published version 3.3.1 with this fix, feel free to comment/re-open if it doesn't work as expected :)
